.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
         exclude: ^(tests/syft/lib/allowlist_report.py)
       - id: requirements-txt-fixer
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.4
+    rev: 3.9.1
```

Comment:
Any specific reason for this change?

Reply:
I haven't made that change. This was done by @gmuraru.

Reply:
hmm..I think you need to rebase? (if this change is on dev, it should not show up in this PR)

Reply:
Agreed, don't try and rebase manually. use git to rebase and pull branch.